homeassistant/components/kodi/strings.json

```diff
@@ -37,7 +37,8 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
-      "unknown": "[%key:common::config_flow::error::unknown%]"
+      "unknown": "[%key:common::config_flow::error::unknown%]",
+      "no_uuid": "Kodi instance does not have a unique id"
```

Comment:
Maybe we should add to this message (the user sees); why this happens and what they can do about it.

Reply:
@frenck Good idea. Is there a way to also pass parameters to the message? That way I can have the ip address in the message, and in case the user has multiple instances I can point at the right one.

Reply:
I don't think so.

But, as far as I understand, this happens on older versions, right?
If so, that could be added as a hint.

Reply:
Yes, that's my plan, but would have been nice to also point at the specific instance.

Reply:
🤦  Nevermind... I just now realized this is about the zeroconf discovery handling. There is no message to the user at this point. The current you have is fine.

Reply:
:)

I already updated the message. Should I revert is the more informative version fine?

Reply:
It is fine 👍